src/main/java/mate/academy/model/ShoppingCart.java

```diff
@@ -0,0 +1,61 @@
+package mate.academy.model;
+
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.MapsId;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import java.util.List;
+
+@Entity
+@Table(name = "shopping_cart")
+public class ShoppingCart {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToMany(cascade = CascadeType.MERGE)
+    private List<Ticket> tickets;
+    @OneToOne//(cascade = CascadeType.ALL)

```

Comment:
It is better to remove
```suggestion
    @OneToOne
```